lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -0,0 +1,70 @@
+<%page args="credit_status" />
+<%!
+    import datetime
+    import pytz
+    from django.utils.translation import ugettext as _
+    from util.date_utils import get_default_time_display
+%>
+<%namespace name='static' file='../static_content.html'/>
+
+% if credit_status["eligible"]:
+    <div class="message message-status is-shown credit-message">
+    % if credit_status["error"]:
+        <p class="message-copy credit-error-msg">
+            ${_(
+                "An unexpected error occurred while retrieving your credit status.  "
```

Comment:
At this point, we will have already charged the user's credit card.  There will be loads of errors being logged at this point, though, so this would be a critical issue that we'd need to immediately address.